packages/tasks/src/tasks/image-text-to-text/data.ts

```diff
@@ -0,0 +1,86 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Instructions composed of image and text.",
+			id: "liuhaotian/LLaVA-Instruct-150K",
+		},
+		{
+			description: "Conversation turns where questions involve image and text.",
+			id: "liuhaotian/LLaVA-Pretrain",
+		},
+		{
+			description: "Screenshots of websites with their HTML/CSS codes.",
+			id: "HuggingFaceM4/WebSight",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "mask-generation-input.png",
+				type: "img",
+			},
+			{
+				label: "Text Prompt",
+				content: "Describe the position of the bee in detail.",
+				type: "text",
+			},
+		],
+		outputs: [
+			{
+				label: "Answer",
+				content:
+					"The bee is sitting on a pink flower, surrounded by other flowers. The bee is positioned in the center of the flower, with its head and front legs sticking out.",
+				type: "text",
+			},
+		],
+	},
+	metrics: [],
+	models: [
```

Comment:
moondream and paligemma might be interesting (replacing some of the current older ones)